### ISSUE TITLE
vg: src/constructor.cpp:2230: void vg::Constructor::construct_graph(const std::vector<FastaReference*>&, const std::vector<vcflib::VariantCallFile*>&, const std::vector<FastaReference*>&, const std::function<void(vg::Graph&)>&): Assertion `reference_for.count(fasta_contig)' failed.

### ISSUE THREAD

**Issue by hndxjsr** [vgteam/vg#4191]
**1. What were you trying to do?**

**2. What did you want to happen?**

**3. What actually happened?**

**4. If you got a line like `Stack trace path: /somewhere/on/your/computer/stacktrace.txt`, please copy-paste the contents of that file here:**

```
Place stacktrace here.
```

**5. What data and command can the vg dev team use to make the problem happen?**

**6. What does running `vg version` say?**

```
Place vg version output here
```

**Comment by jeizenga** [comment]
Based on your issue title, you appear to be running an out-of-date version of VG. If you follow the template we provided for submitting the issue, we might be able to tell you more.

**Comment by faithokamoto** [comment]
Yeah, this is probably fixed in current vg. Or if not, the whole thing is stale since we would've needed more information at the time such as what Jordan asked for.
